deploy/helm/charts/templates/lvm-node-servicemonitor.yaml

```
@@ -0,0 +1,23 @@
+{{- if .Values.lvmNode.serviceMonitor.enabled }}
```

Check if the api is available?

```suggestion
{{- if and ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) .Values.lvmNode.serviceMonitor.enabled }}
```